Makefile

```diff
@@ -11,8 +11,13 @@ default : install
 build : test
```

Comment:
Somewhat unrelated and this was actually in #159, but .gitignore could be updated to ignore the build binary. Alternatively, the build command here could be updated to place the binary in `dist/` which is already ignored. I lean towards adding a new .gitignore entry so `dist/` is mostly just used with goreleaser.

Reply:
Fixed

Reply:
I think dist is fine for now. Thanks for making that change :)